benefits/static/css/styles.css

```diff
@@ -834,6 +834,32 @@ a.card:focus-visible {
   background-image: url("/static/img/modal-trigger--selected.svg");
 }
 
+:root {
+  --alert-box-padding: calc(16rem / 16);
+  --alert-box-border-width: calc(8rem / 16);
+  --alert-box--heading-margin-bottom: calc(4rem / 16);
+  --alert-box--heading-line-height: calc(27rem / 16);
+}
+
+.alert-box {
+  border-left-width: var(--alert-box-border-width);
+  border-left-style: solid;
+  padding: var(--alert-box-padding);
+  color: var(--text-primary-color);
+}
+
+.alert-box a:focus:not(.btn):not(.card):not(.footer-link):not(#skip-to-content),
+a:focus-visible:not(.btn):not(.card):not(.footer-link):not(#skip-to-content) {
```

Comment:
I think there is a missing `.alert-box ` in front of this. Without it, the focus rings for normal links are getting this style applied.

<img src="https://github.com/cal-itp/benefits/assets/25497886/961b2643-3d7d-4590-8f37-acaeb2c7055b" width="400">

<img src="https://github.com/cal-itp/benefits/assets/25497886/8170359d-9260-4a76-9788-f1607a52689f" width="400">

Reply:
Good catch! Fixed in the rebased commit 703bad7b171d261a28b927e519db32ecb094083c